game.rb

```diff
@@ -47,4 +57,19 @@ def draw_background
 
 end
 
-Game.run
+# game = Game.new
+# Thread.new {
+#   game.show
+# }
+# game.draw
+# game.update
+# game.draw
+# puts "Game field: #{game.board.field}"
+
+# game.board.reveal_at({x: 3, y: 0})
+
+# Game.new.show
+
+# while 1
+# end
```

Comment:
I'd still like to be able to run the game in standalone mode. There's an easy option of just checking to see if game.rb was the file that was run, instead of bot.rb...  If so, then run it now. Something like:

```ruby
if ARGV[1] == File.basename(__FILE__)
  Game.run
end
```